scripts/utilities/move_wilderness_images.py

```
@@ -0,0 +1,239 @@
+                    shutil.move(str(img_file), str(original_file))
+                    print(f"  🔄 Moved back: {img_file} → {original_file}")
+                    moved_back += 1
+                except Exception as e:
+                    print(f"  ❌ Error moving back {img_file}: {e}")
+                    kept_images += 1
+        else:
+            kept_images += 1
+
+    # Clean up empty directories
+    removed_dirs = 0
+    for root, dirs, files in wilderness_dir.walk(top_down=False):
```

Ruff (0.12.2)

Loop control variable `dirs` not used within loop body

Rename unused `dirs` to `_dirs`

(B007)